src/sounds.cpp

```diff
@@ -21,7 +21,7 @@
 #ifdef SDL_SOUND
 #include "SDL2/SDL_mixer.h"
 #include <thread>
-#if (defined _WIN32 || defined WINDOWS)
+#if (!defined __MINGW64__ && defined _WIN32 || defined WINDOWS )
```

Comment:
As it's already there, leave it there. Never change a running system, but seriously: it's also used in catacurse.h, cata_tiles.h, cursesdef.h and in many more cpp files. It's either correct or at least doesn't cause any trouble.